ptvsd/wrapper.py

```diff
@@ -2038,6 +2073,7 @@ def on_pydevd_thread_suspend(self, seq, args):
         try:
             vsc_tid = self.thread_map.to_vscode(pyd_tid, autogen=False)
         except KeyError:
+            self.pydevd_notify(pydevd_comm.CMD_THREAD_RUN, pyd_tid)
```

Comment:
Why is this necessary? If `pydevd` is sending some events, why are we calling back into PyDevD?

Reply:
Feels like some workaround/hack for something.

Reply:
This is a case we we got a suspended event before create. This `try` catch was there to handle that issue. But there was another problem. If we don't have a reference for a thread, we would just return from this event. That means, user has no clue why their code is not running. 

I don't think we need to do the check any more, since we have the fix for the thread creation event. I will remove the try/catch. if this fails again then we have bigger problems.